src/js/components/AppListComponent.jsx

```diff
@@ -80,12 +113,42 @@ var AppListComponent = React.createClass({
     });
   },
 
+  getAppsInContext: function () {
```

Comment:
I don't understand that _context_ part of the function naming.
Could this name be more precise?